contrib/unbound.service.in

```diff
@@ -63,8 +63,8 @@ ProtectHome=true
 ProtectControlGroups=true
 ProtectKernelModules=true
 ProtectSystem=strict
-RuntimeDirectory=unbound
-ConfigurationDirectory=unbound
+RuntimeDirectory=@UNBOUND_RUN_DIR@
+ConfigurationDirectory=@UNBOUND_SYSCONF_DIR@
```

Comment:
[UNBOUND_SYSCONF_DIR](https://github.com/NLnetLabs/unbound/blob/a45a7efe694d7524d2d37070b5266453004bd4a1/configure.ac#L161) is the upper system `sysconfdir`, usually `/etc` while `ConfigurationDirectory` expects specific lower dir name beneath.